registration/views.py

```diff
@@ -17,3 +19,28 @@ class FamilyViewSet(
         "post",
     ]
     permission_classes = [permissions.IsAuthenticated]
+
+    @action(
+        methods=["get"],
+        detail=True,
+        permission_classes=[permissions.IsAuthenticated],
+        url_path="students",
+    )
+    def get_students(self, request, pk=None):
+        students = Family.objects.filter(id=pk).first().students
+        return Response(StudentSerializer(students, many=True).data)
+
+
+class StudentViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.CreateModelMixin,
+):
+    queryset = Student.objects.all()
+    serializer_class = StudentSerializer
+    http_method_names = [
+        "get",
+        "post",
+    ]
+    permission_classes = [permissions.IsAuthenticated]
```

Comment:
we actually don't need endpoints specifically for the students resource! we'll later be incorporating the `StudentSerializer` and `FamilyAttendanceSerializer` into other views:
* `StudentSerializer` → GET families-detail
* `FamilyAttendanceSerializer` → GET session-detail

Reply:
True! I removed these lines